src/sentry/static/sentry/app/views/performance/realUserMonitoring/index.tsx

```diff
@@ -0,0 +1,177 @@
+      if (!transactionName) {
+        // If there is no transaction name, redirect to the Performance landing page
+
+        browserHistory.replace({
+          pathname: `/organizations/${organization.slug}/performance/`,
+          query: {
+            ...location.query,
+          },
+        });
+        return null;
+      }
+
+      return (
+        <Redirect
+          router={router}
+          to={transactionSummaryRouteWithQuery({
+            orgSlug: organization.slug,
+            transaction: transactionName,
+            projectID: decodeScalar(location.query.project),
+            query: location.query,
+          })}
+        />
+      );
+    } else {
+      return <Alert type="warning">{t("You don't have access to this feature")}</Alert>;
+    }
+  };
+
+  render() {
+    const {organization, projects, location} = this.props;
+    const {eventView} = this.state;
+    const transactionName = getTransactionName(location);
+    if (!eventView || transactionName === undefined) {
+      // If there is no transaction name, redirect to the Performance landing page
+      browserHistory.replace({
+        pathname: `/organizations/${organization.slug}/performance/`,
+        query: {
+          ...location.query,
+        },
+      });
+      return null;
+    }
+
+    return (
+      <SentryDocumentTitle title={this.getDocumentTitle()} objSlug={organization.slug}>
+        <Feature
+          features={['measurements']}
+          organization={organization}
+          renderDisabled={this.renderNoAccess}
+        >
+          <GlobalSelectionHeader>
+            <StyledPageContent>
+              <LightWeightNoProjectMessage organization={organization}>
+                <RumContent
+                  location={location}
+                  eventView={eventView!}
+                  transactionName={transactionName!}
+                  organization={organization}
+                  projects={projects}
+                />
+              </LightWeightNoProjectMessage>
+            </StyledPageContent>
+          </GlobalSelectionHeader>
+        </Feature>
+      </SentryDocumentTitle>
+    );
+  }
+}
+
+const StyledPageContent = styled(PageContent)`
+  padding: 0;
+`;
+
+function generateRumEventView(
+  location: Location,
+  transactionName: string | undefined
+): EventView | undefined {
+  if (transactionName === undefined) {
+    return undefined;
+  }
+  const query = decodeScalar(location.query.query) || '';
+  const conditions = tokenizeSearch(query);
+  conditions
+    .setTag('event.type', ['transaction'])
+    .setTag('transaction.op', ['pageload'])
+    .setTag('transaction', [transactionName]);
+
+  Object.keys(conditions.tagValues).forEach(field => {
+    if (isAggregateField(field)) conditions.removeTag(field);
+  });
+
+  return EventView.fromNewQueryWithLocation(
+    {
+      id: undefined,
+      version: 2,
+      name: transactionName,
+      fields: ['id', 'user', 'transaction.duration', 'timestamp'],
```

Comment:
Do you want the user tag value, or `user.display`? The `user` tag will often be `id:1235324` style values.

Reply:
Actually, I forgot I can use this `eventView` here. This should be the `p75`s for all the web vitals. Gonna fix this.

Reply:
Addressed in 33a7282